native/ios/Podfile.lock

```diff
@@ -87,7 +87,7 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   boost-for-react-native: 39c7adb57c4e60d6c5479dd8623128eb5b3f0f2c
-  BVLinearGradient: 4b3f3bdf1ac1a52a5ef5e65c9585d793a7c3f825
+  BVLinearGradient: 6d8909e3fc6b089defa4b89d967441fa6827a2ee
```

Comment:
@k0nserv I'm not very well-versed with Cocoapods - is this something we'd expect to see change?

Reply:
FYI this is done automatically when I run `pod install`

Reply:
Like, I feel like it doesn't belong in this PR anyway, as the PR has nothing to do with native.

Reply:
Yeah, it's probably not harmful, but I think it should be reverted anyway for the sake of keeping it out this PR.